src/ReactWrapper.js

```diff
@@ -123,6 +123,9 @@ export default class ReactWrapper {
    * @returns {ReactComponent}
    */
   instance() {
+    if (this.root !== this) {
+      throw new Error('ReactWrapper::instance() can only be called on the root');
```

Comment:
not explicitly...  though it would have thrown before (in a different, less informative way) anyway. This method on anything but the root doesn't make a whole lot of sense